packages/esm-appointments-app/src/helpers/time.tsx

```diff
@@ -19,23 +19,23 @@ export const convertTime12to24 = (time12h, timeFormat: amPm) => {
   return [hours, minutes];
 };
 
-const initialState = { appointmentDate: dayjs(new Date().setHours(0, 0, 0, 0)).format(omrsDateFormat) };
+const initialState = { selectedDate: dayjs(new Date().setHours(0, 0, 0, 0)).format(omrsDateFormat) };
 
-export function getStartDate() {
-  return getGlobalStore<{ appointmentDate: string | Date }>('appointmentStartDate', initialState);
+export function getSelectedDate() {
+  return getGlobalStore<{ selectedDate: string }>('appointment-selected-date', initialState);
```

Comment:
@denniskigen  @ibacher  is there any best practice regarding naming global stores?

Reply:
No... we've been a bit in consistent on this. Does this even need to be in a global store, though? This looks like something that could be kept in a React Context more cleanly (the use case for global stores is really if we need data to be available across different React trees or to survive across a re-render of the whole tree).

Reply:
@ibacher  this was existing each of a global store being used for this... (but I'm happy to try to work into react context if it makes it simpler, will try to give it a go)